nextflow.config

```diff
@@ -49,36 +58,45 @@ params {
     // Downstream analysis
     skip_report = false
     skip_lineage = false
+    // -----------------------------------
+
+    // Reveal specific options -----------
+    collapseby = 'input_id'
+    cloneby = 'subject_id'
+    singlecell='single_cell'
+    outdir = './results'
```

Comment:
I guess this outdir should be removed, because it is defined later in the boilerplate options